### ISSUE TITLE
The loss does not work in ssd

### ISSUE THREAD

**Issue by MisayaZ** [libuyu/GHM_Detection#10]
Hi 
I rewrite your code with tensorflow and use these two loss in ssd object detection framework. However, the training can not converge. The loss value is very small at the beginning and did not become small after many iteration.
I just add GHMC_LOSS and GHMR_LOSS and regard the sum as the final loss for training. Should I do something else to GHMC_LOSS and GHMR_LOSS? What step have I ignored?

**Comment by ShangxuanWu** [comment]
@MisayaZ  I have the same problem with you. I only used GHM-R loss on boxes. Comparing to smooth_l1 regression loss, this is significantly worse. Can I get your wechat? We can talk more about this.

**Comment by ShangxuanWu** [comment]
@MisayaZ Are you using the momentum-ed version or vanilla GHM-R?

**Comment by libuyu** [comment]
It seems that both Focal Loss and GHM Losses do not work well if directly applied in SSD. Maybe it has some special properties (like the atrous conv) that need specialized hyper-parameter. We will look into it.

**Comment by ShangxuanWu** [comment]
Hi @libuyu , I made it work using momentum 0.5 and mu 0.02 in large datasets, in my tensorflow implementation. Thanks!

**Comment by xiaowei-ui** [comment]
> Hi @libuyu , I made it work using momentum 0.5 and mu 0.02 in large datasets, in my tensorflow implementation. Thanks!

Excuse me, when you change the momentum, how does it add GHM loss to the SSD model?thanks!

**Comment by immarioo** [comment]
> 
> 
> > Hi @libuyu , I made it work using momentum 0.5 and mu 0.02 in large datasets, in my tensorflow implementation. Thanks!
> 
> Excuse me, when you change the momentum, how does it add GHM loss to the SSD model?thanks!

hi, have you add the ghm loss to the ssd model, could you please tell me some details. thanks.
